chunky/src/java/se/llbit/chunky/chunk/biome/BiomeData.java

```diff
@@ -0,0 +1,33 @@
+package se.llbit.chunky.chunk.biome;
+
+import se.llbit.chunky.world.Chunk;
+
+/**
+ * Interface designed to allow for any biome data structure behind it
+ *
+ * Implementations expected to handle x/z values from 0-15
+ * Implementations expected to gracefully handle ANY y values, though it does not need to support all values
+ *
+ * If a given y coordinate is outside of supported bounds, the implementation should return 0
+ */
```

Comment:
It would be better if the documentation of what the methods should return would be on the methods. The interface documentation should tell what the interface does and the general idea (i.e. only the first sentence of this).